Xamarin.Forms.Platform.iOS/VisualElementPackager.cs

```diff
@@ -45,7 +45,23 @@ public void Load()
 			{
 				var child = ElementController.LogicalChildren[i] as VisualElement;
 				if (child != null)
+				{
 					OnChildAdded(child);
+
+					if (!CompressedLayout.GetIsHeadless(child))
+					{
+						var childRenderer = Platform.GetRenderer(child);
```

Comment:
I'm wondering if it would be possible to lift this code and put it into something common so it can be used here as well as in `EnsureChildrenOrder` to reduce duplicated code?

Besides from that I think this is a nice performance win!